modules/packages/Sort.chpl

```diff
@@ -1241,7 +1247,7 @@ module QuickSort {
         var mid = lo + (hi - lo + 1) / 2;
         var piv = mid;
 
-        if hi - lo < 0 { // minlen {
+        if hi - lo < minlen { // minlen {
```

Comment:
Probably doesn't need the comment any more?

Reply:
No worries, good suggestion, I've fixed it